### PR TITLE
Config tweaks and one variable fix

### PR DESCRIPTION
**Pull request by cafreeman**
I saw that you'd made some of the changes we talked about in the PR, but there were a couple more config tweaks that needed to happen to make everything work, namely adding a specific babel plugin to the mix.

Also, when I booted the app I found that there was an issue with a couple variable references in one of the React components so I fixed those!

### CODE REVIEW

File: apps/web/assets/js/components/Opportunity.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default class Opportunity extends React.Component {
                 <a href={project.url}>{project.name}</a> 
                 <span className="opportunity__title-icon">&#9655;</span>
               </span>
-              <a href={url}>{title}</a> 
+              <a href={project.url}>{project.title}</a>
```
Comment (doomspork) [inline apps/web/assets/js/components/Opportunity.jsx]: I _think_ this is incorrect.  This should be the opportunities `url` and `title`, not the `project` which is above.

File: apps/web/assets/brunch-config.js [hunk 2]
```diff
@@ -26,7 +26,10 @@ exports.config = {
   // Configure your plugins
   plugins: {
     babel: {
-      presets: ['latest', 'react']
+      presets: ['latest', 'react'],
+      plugins: [
+        "transform-class-properties"
```
Comment (doomspork) [inline apps/web/assets/brunch-config.js]: For my own edification could you tell me what this does?  Also, should we use single quotes for consistency or is double necessary here?

File: apps/web/assets/js/components/Opportunity.jsx (hunk 1 above)
Comment (cafreeman) [reply apps/web/assets/js/components/Opportunity.jsx]: Ohhh, ok. Do those exist elsewhere? Those two variables were undefined and throwing errors when I first booted the app

File: apps/web/assets/brunch-config.js (hunk 2 above)
Comment (cafreeman) [reply apps/web/assets/brunch-config.js]: Oh, nope. You can use single quotes for sure. 

This adds a specific Babel plugin for supporting the instance methods we were talking about earlier. Babel is actually just a big bundle of plugins, each of which address a certain JS feature you want to transpile.

Presets are just bundles of commonly used plugins (for the sake of convenience), but sometimes you have to pull in specific plugins as well

Comment (cafreeman) [reply apps/web/assets/brunch-config.js]: Oh, nope. You can use single quotes for sure. 

This adds a specific Babel plugin for supporting the instance methods we were talking about earlier. Babel is actually just a big bundle of plugins, each of which address a certain JS feature you want to transpile.

Presets are just bundles of commonly used plugins (for the sake of convenience), but sometimes you have to pull in specific plugins as well

File: apps/web/assets/js/components/Opportunity.jsx (hunk 1 above)
Comment (cafreeman) [reply apps/web/assets/js/components/Opportunity.jsx]: Maybe they need to be destructured off a different part of props?

Comment (doomspork) [reply apps/web/assets/js/components/Opportunity.jsx]: Hmm 🤔

This is the pre-refactored code:

```javascript
 <a href={this.props.data.url}>{this.props.data.title}</a> 
```

I'll try it again locally 👍

File: apps/web/assets/brunch-config.js (hunk 2 above)
Comment (doomspork) [reply apps/web/assets/brunch-config.js]: Thanks @cafreeman 😁

File: apps/web/assets/js/components/Opportunity.jsx (hunk 1 above)
Comment (cafreeman) [reply apps/web/assets/js/components/Opportunity.jsx]: ahhhh, got it. It looks like the issue was actually just the `url` wasn't getting pulled out of `this.props.data`.

Fix incoming

**Comment by doomspork** [comment]
Thank you @cafreeman 👍

**Pull request merged**